src/Core/Settings.h

```diff
@@ -351,6 +351,8 @@ class IColumn;
     M(Bool, opentelemetry_trace_processors, false, "Collect OpenTelemetry spans for processors.", 0) \
     M(Bool, prefer_column_name_to_alias, false, "Prefer using column names instead of aliases if possible.", 0) \
     M(Bool, allow_experimental_analyzer, false, "Allow experimental analyzer", 0) \
+    M(Bool, allow_experimental_get_http_header, false, " experimental function getHTTPHeader", 0) \
```

Comment:
no need to call it experimental, the boolean is needed just as protection
`allow_get_client_http_header` sounds okay and change the description